include/jemalloc/internal/jemalloc_internal_types.h.in

```diff
@@ -33,17 +33,18 @@ typedef enum malloc_init_e malloc_init_t;
  *
  * a: arena
  * t: tcache
+ * .: dependent on --with-lg-tcache-limit
```

Comment:
Nit: maybe make explicit in the comment this is either (a)rena or (t)cache - at the moment it's not clear what `.` is actually going to be used for.

Reply:
Done, thanks.